### PR TITLE
Update rules for Kym-cdn.com

### PR DESCRIPTION
**Pull request by Nintendofan885**
Add s.kym-cdn.com and i.kym-cdn.com subdomains and add test urls for the others
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring existing code
- [ ] New Ruleset
- [x] Existing Ruleset

## For Rulesets: Testing

- [x] Travis CI completed without errors (triggered upon pull request)

### CODE REVIEW

File: src/chrome/content/rules/KYM-CDN.com.xml [hunk 1]
```diff
@@ -11,10 +11,23 @@
 -->
 <ruleset name="KYM-CDN.com (partial)" default_off="mismatched">
```
Comment (J0WI) [inline src/chrome/content/rules/KYM-CDN.com.xml:12]:
```suggestion
<ruleset name="KYM-CDN.com">
```

File: src/chrome/content/rules/KYM-CDN.com.xml [hunk 2]
```diff
@@ -11,10 +11,23 @@
...
 
+	<target host="i.kym-cdn.com" />
+		<test url="http://i.kym-cdn.com/entries/icons/mobile/000/006/363/pokemon-balls-pokeball-1920x1080-5166.jpg/" />
```
Comment (J0WI) [inline src/chrome/content/rules/KYM-CDN.com.xml:15]:
```suggestion
		<test url="http://i.kym-cdn.com/entries/icons/mobile/000/006/363/pokemon-balls-pokeball-1920x1080-5166.jpg" />
```

File: src/chrome/content/rules/KYM-CDN.com.xml [hunk 3]
```diff
@@ -11,10 +11,23 @@
...
+	
 	<target host="i0.kym-cdn.com" />
+		<test url="http://i0.kym-cdn.com/entries/icons/mobile/000/006/363/pokemon-balls-pokeball-1920x1080-5166.jpg/" />
```
Comment (J0WI) [inline src/chrome/content/rules/KYM-CDN.com.xml:18]:
```suggestion
		<test url="http://i0.kym-cdn.com/entries/icons/mobile/000/006/363/pokemon-balls-pokeball-1920x1080-5166.jpg" />
```

File: src/chrome/content/rules/KYM-CDN.com.xml [hunk 4]
```diff
@@ -11,10 +11,23 @@
...
+	
 	<target host="i1.kym-cdn.com" />
+		<test url="http://i1.kym-cdn.com/entries/icons/mobile/000/006/363/pokemon-balls-pokeball-1920x1080-5166.jpg/" />
```
Comment (J0WI) [inline src/chrome/content/rules/KYM-CDN.com.xml:21]:
```suggestion
		<test url="http://i1.kym-cdn.com/entries/icons/mobile/000/006/363/pokemon-balls-pokeball-1920x1080-5166.jpg" />
```

File: src/chrome/content/rules/KYM-CDN.com.xml [hunk 5]
```diff
@@ -11,10 +11,23 @@
...
+	
 	<target host="i2.kym-cdn.com" />
+		<test url="http://i2.kym-cdn.com/entries/icons/mobile/000/006/363/pokemon-balls-pokeball-1920x1080-5166.jpg/" />
```
Comment (J0WI) [inline src/chrome/content/rules/KYM-CDN.com.xml:24]:
```suggestion
		<test url="http://i2.kym-cdn.com/entries/icons/mobile/000/006/363/pokemon-balls-pokeball-1920x1080-5166.jpg" />
```

File: src/chrome/content/rules/KYM-CDN.com.xml [hunk 6]
```diff
@@ -11,10 +11,23 @@
...
+	
 	<target host="i3.kym-cdn.com" />
+	<test url="http://i3.kym-cdn.com/entries/icons/mobile/000/006/363/pokemon-balls-pokeball-1920x1080-5166.jpg/" />
```
Comment (J0WI) [inline src/chrome/content/rules/KYM-CDN.com.xml:27]:
```suggestion
	<test url="http://i3.kym-cdn.com/entries/icons/mobile/000/006/363/pokemon-balls-pokeball-1920x1080-5166.jpg" />
```

File: src/chrome/content/rules/KYM-CDN.com.xml [hunk 7]
```diff
@@ -11,10 +11,23 @@
...
+	
+	<target host="s.kym-cdn.com" />
+		<test url="http://s.kym-cdn.com/assets/kym-logo-fbdeb4f48d4b9e0d7f9675fa5e1e011a.png/" />
```
Comment (J0WI) [inline src/chrome/content/rules/KYM-CDN.com.xml:30]:
```suggestion
		<test url="http://s.kym-cdn.com/assets/kym-logo-fbdeb4f48d4b9e0d7f9675fa5e1e011a.png" />
```

**Comment by Nintendofan885** [comment]
@J0WI You should of probably done all the changes removing the slashes at the test urls in the same bit to make it easier to commit

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**